docs/sources/enterprise/access-control/_index.md

```diff
@@ -25,14 +25,19 @@ To grant or revoke access to your users, create or remove built-in role assignme
 
 ## Resources with fine-grained permissions
 
-Fine-grained access control is currently available for [Reporting]({{< relref "../reporting.md" >}}) and [Managing Users]({{< relref "../../manage-users/_index.md" >}}).
-To learn more about specific endpoints where you can use access control, refer to [Permissions]({{< relref "./permissions.md" >}}) and to the relevant API guide:
-
-- [Fine-grained access control API]({{< relref "../../http_api/access_control.md" >}})
-- [Admin API]({{< relref "../../http_api/admin.md" >}})
-- [Organization API]({{< relref "../../http_api/org.md" >}})
-- [Reporting API]({{< relref "../../http_api/reporting.md" >}})
-- [User API]({{< relref "../../http_api/user.md" >}})
+Fine-grained access control is currently available for the following capabilities:
```

Comment:
```suggestion
Fine-grained access control is available for the following capabilities:
```
"Currently" is covered by the version number, so that term is redundant in nature.